### PR TITLE
Conversation list null states

### PR DESCRIPTION
**Pull request by saulmc**
- Show matchmaker recommendations for new users with onchain recommendations.
- Show QR code for users with no onchain recommendations.
- Show beta chat invite.

### CODE REVIEW

**Comment by lourou** [comment]
Looks good @saulmc! 

Two comments:

1. The Share Link is cut off on the iPhone mini due to the reduced screen height on the mini models.

2. When I accept the invite on the web page to join the private beta group, the spinner animates for a few seconds and then displays "An error occurred."

### Screenshots

**Comment by saulmc** [comment]
Thx @lourou , fixed!

File: config.ts [hunk 1]
```diff
@@ -68,7 +68,8 @@ const ENV = {
       (domain) => [`https://${domain}`, `http://${domain}`, domain]
     ),
     enableTransactionFrames: true,
-    betaGroupChatUrl: "https://converse.xyz/group-invite/NfrIwpnhZdRCx2ZyGCg-S",
+    alphaGroupChatUrl:
+      "https://converse.xyz/group-invite/NfrIwpnhZdRCx2ZyGCg-S",
```
Comment (cameronvoell) [inline config.ts:72]: this url shows up as not found, not sure if it was supposed to be updated as well

**Review by cameronvoell — APPROVED** [review]
See comment that theres is one link that didnt seem to be working. But otherwise updates look good. cool improvement!

**Comment by darickdang** [comment]
Here's an initial pass.

I think the "Help shape the future of Converse!" gets lost when stuck at the bottom so I pushed it to the top and made it so it could be dismissed. It may be a useful component to use in the future for other callouts/announcements as well.

#730 - Conversation list null states

**Pull request merged**